### PR TITLE
enable containers/buildah and mergebot

### PR DESCRIPTION
**Pull request by baude**
the containers/buildah team would like to use the openshift merge bot for their repository.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -1986,15 +1986,6 @@ external_plugins:
     - pull_request
     name: needs-rebase
   containers/buildah:
-  - endpoint: http://refresh
```
Comment (alvaroaleman) [inline core-services/prow/02_config/_plugins.yaml:1989]: why did you remove these?

Comment (baude) [reply core-services/prow/02_config/_plugins.yaml:1989]: we dont use them in other containers/ repos.  I figured I would remove cruft.  Would you advice differ?

**Review by hongkailiu — APPROVED** [review]
